.github/workflows/ci.yml

```diff
@@ -162,3 +163,14 @@ jobs:
       - name: Push Multi-Platform Package to Upbound
         if: env.XPKG_ACCESS_ID != ''
         run: "./crossplane --verbose xpkg push --package-files $(echo *.xpkg|tr ' ' ,) ${{ env.XPKG }}:${{ env.XPKG_VERSION }}"
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3.3.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
i don't have permissions to see the github actions settings, but I assume the default token has been granted package write permissions?

Reply:
Yes, a package was released after the PR merge: https://github.com/crossplane-contrib/function-auto-ready/pkgs/container/function-auto-ready